client/app/components/group/group.spec.js

```diff
@@ -65,8 +65,10 @@ describe("Group", () => {
   describe("Route", () => {
     beforeEach(() => {
       $httpBackend.whenGET("/api/auth/status/").respond({ id: 43 });
-      inject(($translate, $q) => {
+      inject(($translate, $q, $injector) => {
```

Comment:
Can't you simply write `inject(($translate, $q, CurrentGroup) => {` here instead of the verbose `injector.get` syntax?